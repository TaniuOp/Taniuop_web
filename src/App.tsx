import React from 'react';
import MainContainer from './components/PortfolioSections/MainContainer';
import Footer from './components/PortfolioSections/Footer';
import { BrowserRouter } from 'react-router-dom';
import './styles/styles.scss';

function App() {
  return (
    <>
      <div className='App'>
        <BrowserRouter>
          <MainContainer />
        </BrowserRouter>
      </div>
      <div className='Footer'>
        <Footer />
      </div>
    </>
  );
}

export default App;
