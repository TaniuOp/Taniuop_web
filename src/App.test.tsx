import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

test('App', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
