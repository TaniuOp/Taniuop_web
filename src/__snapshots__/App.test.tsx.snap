// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 1`] = `
<DocumentFragment>
  <div
    class="App"
  />
  <div
    class="Footer"
  >
    <h2>
      Footer
    </h2>
  </div>
</DocumentFragment>
`;
